### PR TITLE
Update perform-G-Suite-migration.md

### PR DESCRIPTION
**Pull request by sulobr**
Added clarification on line 235. Seems like customers are not clear that the email address here should be the G suite Admin account and not the service account they just created.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ We recommend that the primary address (sometimes referred to as the "User Id") f
 
 6. After selecting the CSV file, click **Open**. Back on the **new migration batch** page, click **Next**.
 
-7. Enter the email address for the super admin within the G Suite environment. This email address will be used to test connectivity between G Suite and Microsoft 365 or Office 365.
+7. Enter the email address for the super admin within the G Suite environment. This is not the service account you just created, it should be the email address of the GSuite Admin. This email address will be used to test connectivity between G Suite and Microsoft 365 or Office 365.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md:235]:
```suggestion
7. Enter the email address for the super admin within the G Suite environment. This is not the service account you just created, it should be the email address of the G Suite admin. This email address will be used to test connectivity between G Suite and Microsoft 365 or Office 365.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joe429mama** [comment]
Ok

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**